src/Core/SqlManager/Exception/SqlManagerExportException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\SqlManager\Exception;
+
+use RuntimeException;
+
+class SqlManagerExportException extends RuntimeException
+{
+    const SQL_REQUEST_ERROR = 1;
+    const SQL_REQUEST_HAS_NO_DATA = 2;
```

Comment:
`SQL_REQUEST_NO_DATA` otherwise do the same for `SQL_REQUEST_HAS_ERROR` (but I prefer without the `HAS`)

Reply:
maybe `SQL_REQUEST_EMPTY_EXECUTION_RESULT` :thinking: ? a bit too long, but clear enough?

Reply:
`SQL_REQUEST_EMPTY_RESULT`? It's an ExportException so it's obviously during execution no?